### PR TITLE
Support bevy 0.15

### PR DESCRIPTION
**Pull request by ActuallyHappening**

### CODE REVIEW

**Comment by ActuallyHappening** [comment]
The examples take a long time to port over to using required components, but the main crate compiles atm

**Comment by ActuallyHappening** [comment]
Currently depending on my port of transform-gizmo to work though, waiting on them to do a crates-io release

**Comment by jakobhellermann** [comment]
thanks! I'm gonna go ahead and merge this, I can remove the patch when the transform-gizmo crate is released. Can you write a comment here if you notice before me?

**Comment by jakobhellermann** [comment]
one thing that's broken right now is the `ui` example, the UI only gets rendered to the regular game camera.

The mechanism by which the "UI" checkbox worked has been changed, as far as I can tell the logic now renders UI to
- the camera tagged `IsDefaultUiCamera`
- the `TargetCamera` attached to the root node

So either we have to find a way to render UI to multiple cameras, or manually switch the targetcamera or isdefaultuicamera in a way that makes sense.

**Comment by jakobhellermann** [comment]
and it would be neat to integrate with the upstreamed picking support, but that's not a release blocker

**Comment by ActuallyHappening** [comment]
Yeah inbuilt picking support is really cool, but not necessary atm for what I do
About the changes to UI rendering to cameras, yeah I noticed that for bevy_cosmic_edit as well, but tbh bevy_editor_pls being buggy when switching to editor cameras is kinda what I expect at this point. A problem for `bevy` to solve internally when it eventually gets its official editor

File: Cargo.toml [hunk 1]
```diff
@@ -3,23 +3,25 @@ members = ["crates/*"]
...
 
-bevy-inspector-egui = "0.26.0"
-egui = "0.28"
-egui_dock = "0.13"
-# used to be egui-gizmo 0.16
-transform-gizmo-bevy = "0.3"
+bevy-inspector-egui = "0.28.0"
+egui = "0.29"
+egui_dock = "0.14"
+transform-gizmo-bevy = "0.5"
```
Comment (SKY-ALIN) [inline Cargo.toml:21]: Version 0.5 doesn't exist

**Pull request merged**